src/audio/codec_adapter/codec_adapter.c

```diff
@@ -401,80 +401,40 @@ int codec_adapter_copy(struct comp_dev *dev)
 
 static int codec_adapter_set_params(struct comp_dev *dev, struct sof_ipc_ctrl_data *cdata)
 {
-	int ret;
-	char *dst, *src;
-	static uint32_t size;
-	uint32_t offset;
 	struct processing_module *mod = comp_get_drvdata(dev);
 	struct module_data *md = &mod->priv;
+	enum module_cfg_fragment_position pos;
+	uint32_t data_offset_size;
+	static uint32_t size;
```

Comment:
both `size_t`?

Reply:
any reason against `size_t`?

Reply:
the API uses uint32_t. So I prefer to keep this the same too